apps/test/condBlockTest.js

```diff
@@ -0,0 +1,136 @@
+var testUtils = require('./util/testUtils');
+var assert = testUtils.assert;
+var canvas = require('canvas');
+
+global.Image = canvas.Image;
+global.Turtle = {};
+
+// needed for Hammerjs
+global.navigator = {};
```

Comment:
If this gets done in other test files, could we potentially do something like add a `testUtils.installHammerStubs()`, or wedge this stubbing in `setupTestBlockly()`?